### PR TITLE
Allow devcloud to work

### PR DESCRIPTION
**Pull request by hdelan**
Some fixes needed to allow intel devcloud to work. Also have added a sample `sub.sh` so that job submission is easy

### CODE REVIEW

File: Code_Exercises/Exercise_02_Hello_World/README.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ Then use the stream you constructed within the SYCL kernel function to print
 
 For DPC++ (using the Intel DevCloud):
 ```sh
+dpcpp -fsycl -o sycl-ex-2 -I../External/Catch2/single_include ../Code_Exercises/Exercise_02_Hello_World/source.cpp
```
Comment (dkhaldi) [inline Code_Exercises/Exercise_02_Hello_World/README.md:53]: we don't need -fsycl if using dpcpp

File: Code_Exercises/Exercise_02_Hello_World/solution.cpp [hunk 2]
```diff
@@ -11,7 +11,11 @@
 #define CATCH_CONFIG_MAIN
 #include <catch2/catch.hpp>
 
+#if __has_include(<sycl/sycl.hpp>)
 #include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
+#endif
```
Comment (dkhaldi) [inline Code_Exercises/Exercise_02_Hello_World/solution.cpp:18]: sorry you had to move this back.
I am not sure which version of DPCPP is still using the old style. But keeping this is safer anyway so we become independent from the implementation.

File: Code_Exercises/Exercise_02_Hello_World/README.md (hunk 1 above)
Comment (hdelan) [reply Code_Exercises/Exercise_02_Hello_World/README.md]: Thanks for this, have updated READMEs

File: Code_Exercises/Exercise_02_Hello_World/solution.cpp (hunk 2 above)
Comment (hdelan) [reply Code_Exercises/Exercise_02_Hello_World/solution.cpp:18]: No prob

**Review by dkhaldi — APPROVED** [review]
LGTM

**Pull request merged**